phcnb/cli.py

```diff
@@ -69,7 +69,7 @@ def create_input_from_param(parameter: Parameter):
 @click.argument('notebook', required=True)
 @click.argument('workflow_output', required=True, type=click.File('w'))
 def workflow(notebook, workflow_output, image, notebook_tool):
-    """ Convert NOTEBOOK into a workflow writton to WORKFLOW_OUTPUT.
```

Comment:
This help string is referring to the `notebook` argument, so it should stay as `NOTEBOOK`.

Reply:
Done